api/v1beta1/common_types.go

```diff
@@ -16,10 +16,35 @@ limitations under the License.
 
 package v1beta1
 
-// Hash - struct to add hashes to status
-type Hash struct {
-	// Name of hash referencing the parameter
-	Name string `json:"name,omitempty"`
-	// Hash
-	Hash string `json:"hash,omitempty"`
+// PasswordSelector to identify the DB and AdminUser password from the Secret
+type PasswordSelector struct {
```

Comment:
could add this as a common type to lib-common for all operators, but then we'd have to set the defaults via webhook

Reply:
Let's think about this in follow-up PRs and future work if that's okay with you

Reply:
+1